constants/appLinks.ts

```diff
@@ -77,6 +79,16 @@ const TEXTBOOK_DEMO_START_LEARNING: GeneralLink = {
   }
 }
 
+const textbookDemoIntroductionCourseStartLearningUrl = 'https://project-platypus-default.apis-dev-5dd5718798d097eccc65fac4e78a33ce-0000.us-south.containers.appdomain.cloud/course/learning-states-and-qubits/introduction'
+
+const TEXTBOOK_DEMO_INTRODUCTION_COURSE_START_LEARNING: GeneralLink = {
```

Comment:
Please, move this link to the component you use it

Reply:
Are we starting to do this for all cases? I thought we were delaying this until https://github.com/Qiskit/qiskit.org/issues/1566. 😕